src/components/Header.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+function Header() {
```

Comment:
Perhaps rename it to `HomepageBanner` as it will make it more clear what this component is.

Reply:
Addressed. The component name has been changed to `HomepageBanner`. Thanks!